lineman/app/components/invitation_form/invitation_form.coffee

```diff
@@ -1,98 +1,18 @@
 angular.module('loomioApp').factory 'InvitationForm', ->
   templateUrl: 'generated/components/invitation_form/invitation_form.html'
-  controller: ($scope, $rootScope, $q, group, Records, CurrentUser, AbilityService, LoadingService, FlashService) ->
+  controller: ($scope, group, CurrentUser, AbilityService, FlashService, RestfulClient, ModalService, TeamLinkModal, AddMembersModal) ->
     $scope.group = group
-    $scope.invitations = []
-    $scope.messageFieldHidden = true
+    $scope.form = { emailAddresses: '' }
+    $scope.showCustomMessageField = false
+    $scope.isDisabled = false
+    $scope.noInvitations = false
+    remote = new RestfulClient('invitations')
```

Comment:
Shouldn't we have a records interface of some kind instead of manually creating a restful client in the controller?